pom.xml

```diff
@@ -250,6 +250,14 @@
             <version>${jmh.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.openjdk.jmh</groupId>
+            <artifactId>jmh-generator-annprocess</artifactId>
+            <version>${jmh.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
You are adding this.

Reply:
Oh, what's wrong with this dependency?

I didn't manage to run the micro benchmark without this module.

Reply:
Take a close look at the CI error log, you posted to me. You are the one taking care of new things.

I am not saying anything is wrong, I am pointing you out, your changes trigger errors.

Reply:
> You are adding this.

I see what's wrong here. I am trying to resolve this.

Reply:
Fixed